.github/workflows/ci.yml

```diff
@@ -31,24 +39,74 @@ jobs:
         run: ./build_test_binaries.sh
         working-directory: air/tests/test_module
 
-      - name: Run cargo fmt
-        run: cargo fmt --all -- --check
-
-      - name: Run cargo check
-        run: cargo check
-
-      - name: Check native air-trace
-        run: cargo check --package air-trace --no-default-features
+      - name: Run cargo build
+        run: cargo build
 
       - name: Run cargo test
+        env:
+          # required by grcov
+          CARGO_INCREMENTAL: 0
+          RUSTFLAGS: "-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests -Cpanic=abort"
+          RUSTDOCFLAGS: "-Cpanic=abort"
         run: |
           cargo test --release
           # Check that it does compile
           cargo bench --no-run
           # The `memory` sanitizer on cargo test has false positive even on empty project.
           for san in address leak; do
-              RUSTFLAGS="-Z sanitizer=$san" cargo test --features test_with_native_code --target x86_64-unknown-linux-gnu
+            RUSTFLAGS="$RUSTFLAGS -Z sanitizer=$san" cargo test --features test_with_native_code --target x86_64-unknown-linux-gnu
           done
 
+      - name: Setup grcov
+        uses: SierraSoftworks/setup-grcov@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          version: latest
+
+      - name: Run grcov
+        id: coverage
+        uses: actions-rs/grcov@v0.1
```

Comment:
Is it OK that grcov collects data from multiple runs?
It should to get data from `cargo test --features test_with_native_code` only.  I don't know if sanitizer affect coverage report, probably not.